modules/globebrowsing/globes/pointglobe.cpp

```diff
@@ -57,20 +68,26 @@ bool PointGlobe::initialize() {
     glBindVertexArray(_vaoID);
 
     // Vertex data is only one point in the origin
-    glm::vec3 data = glm::vec3(0,0,0);
+    std::vector<glm::vec2> data;
```

Comment:
Yes, I'm pretty sure these types are guaranteed not to store any other data. https://github.com/g-truc/glm/issues/225